protos/flyteidl/admin/project_domain_attributes.proto

```diff
@@ -23,3 +23,29 @@ message ProjectDomainAttributesUpdateRequest {
 // Purposefully empty, may be populated in the future.
 message ProjectDomainAttributesUpdateResponse {
 }
+
+message ProjectDomainAttributesGetRequest {
```

Comment:
We should either allow people to select the resource type (the one of thing in MatchingAttibute) or return a list of attributes... right?

Reply:
wow yes, thank you for the catch!

Reply:
done